nss-day-cohort-41/react-nutshell-thepecans
# Ba registration

# Description
Registration form is fully functional.  checks against users in database for duplicate email or username and verifies password again conformation password.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Register a user and verify that they appear in the database.  Verify that you cannot add an existing user and that your password needs to match confirmation password.
# Checklist:
